lib/pipefy_message/providers/aws_client/sns_broker.rb

```diff
@@ -28,23 +28,33 @@ def initialize(_opts = {})
         ##
         # Publishes a message with the given payload to the SNS topic
         # with topic_name.
-        def publish(payload, topic_name, context = nil)
+        def publish(payload, topic_name, context = nil, correlation_id = nil)
           context = "NO_CONTEXT_PROVIDED" if context.nil?
+          correlation_id = "NO_CID_PROVIDED" if correlation_id.nil?
+          event_id = SecureRandom.uuid.to_s
```

Comment:
I know I added this in a bad place, I should've used the higher-lever `Publisher` class :sweat_smile: Brain fart, I'll fix this on Monday